Buildings/Fluid/Air/BaseClasses/ElectricHeater.mo

```diff
@@ -0,0 +1,149 @@
+within Buildings.Fluid.Air.BaseClasses;
+model ElectricHeater "Model for electric heater"
+  extends Buildings.Fluid.Interfaces.PartialTwoPortInterface;
+  extends Buildings.Fluid.Interfaces.TwoPortFlowResistanceParameters(
+    final computeFlowResistance=(abs(dp_nominal) > Modelica.Constants.eps));
+
+  parameter Modelica.SIunits.Efficiency eta(max=1) = 1
+  "Effciency of electrical heater";
+  parameter Modelica.SIunits.HeatFlowRate QMax_flow(min=0) = Modelica.Constants.inf
+    "Maximum heat flow rate for heating (positive)"
+    annotation (Evaluate=true);
+  parameter Modelica.SIunits.Temperature T_start=Medium.T_default
+    "Start value of temperature"
+    annotation(Dialog(tab = "Initialization"));
+   // Dynamics
+  parameter Modelica.Fluid.Types.Dynamics energyDynamics=Modelica.Fluid.Types.Dynamics.SteadyState
+    "Type of energy balance: dynamic (3 initialization options) or steady state"
+    annotation(Evaluate=true, Dialog(tab = "Dynamics", group="Equations"));
+  parameter Modelica.SIunits.Time tau(min=0) = 10
+    "Time constant at nominal flow rate (used if energyDynamics or massDynamics not equal Modelica.Fluid.Types.Dynamics.SteadyState)"
+    annotation(Dialog(tab = "Dynamics"));
+  parameter Boolean homotopyInitialization = true "= true, use homotopy method"
+    annotation(Evaluate=true, Dialog(tab="Advanced"));
+
+  Modelica.Blocks.Interfaces.RealOutput Q_flow(quantity="HeatFlowRate", unit="W")
+    "Heat flow rate added to the fluid (if flow is from port_a to port_b)"
+    annotation (Placement(transformation(extent={{100,70},{120,90}})));
+  Modelica.Blocks.Interfaces.RealOutput P(quantity="Power", unit="W")
```

Comment:
Chiller model such as `Buildings.Fluid.Chillers.ElectricEIR` and mover model such as `Buildings.Fluid.Movers.FlowControlled_dp` use `P` to represent power instead of `pow`.